src/common/pages/home/components/PasswordButtons/component.jsx

```diff
@@ -1,34 +1,42 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faShareAlt, faPlus } from '@fortawesome/free-solid-svg-icons';
 import { Button } from 'antd';
 
 import { StyledButtons, StyledIcon } from './styles';
 
-const PasswordButtons = props => {
-  const { isActiveShareBtn, goToEditPage } = props;
-  return (
-    <StyledButtons>
-      <Button size="large" type="primary" onClick={() => goToEditPage()}>
-        Add
-        <StyledIcon>
-          <FontAwesomeIcon icon={faPlus} />
-        </StyledIcon>
-      </Button>
-      <Button disabled={!isActiveShareBtn} size="large" type="primary">
-        Share
-        <StyledIcon>
-          <FontAwesomeIcon icon={faShareAlt} />
-        </StyledIcon>
-      </Button>
-    </StyledButtons>
-  );
-};
+class PasswordButtons extends PureComponent {
+  handleAddBtnClick = () => {
+    const { goToAddPage } = this.props;
+
+    goToAddPage();
+  };
+
+  render() {
+    const { isActiveShareBtn } = this.props;
+    return (
+      <StyledButtons>
+        <Button size="large" type="primary" onClick={this.handleAddBtnClick}>
+          Add
```

Comment:
May be it make sense to wrap this text into `<span/>`?

Reply:
I think it is not required